tests/unit_tests/simtel/test_simtel_io_metadata.py

```diff
@@ -74,3 +75,117 @@ def test_get_sim_telarray_telescope_id_to_telescope_name_mapping(sim_telarray_fi
 
     assert tel_mapping[1] == "LSTN-01"
     assert tel_mapping[5] == "MSTN-01"
+
+
+def test_get_telescope_list_from_input_card_parses_telescopes(monkeypatch):
+    simtel_io_metadata._get_telescope_list_from_input_card.cache_clear()
+
+    # Simulate InputCard object with parse() returning a string matching the regex
+    class FakeInputCard:
+        def parse(self):
+            return b"""
+                TELESCOPE    -70.91E2     -52.35E2 45.00E2  12.50E2  # (ID=1)  LSTN   01   2B5\n
+                TELESCOPE   -153.29E2     168.86E2 28.70E2  9.15E2  # (ID=6)   MSTN   02   4B1\n
```

Comment:
I believe the regex should work, but do you want to add maybe a test for the other telescope types as well?